website/project/model.py

```diff
@@ -4092,15 +4094,28 @@ def _email_template_context(self, user, is_authorizer=False, urls=None):
 
     def _add_success_logs(self, node, user):
         src = node.registered_from
+        submitted_time = ''
+
+        try:
+            draft = DraftRegistration.find_one(Q("registered_node", "eq", node))
+            # lazy log choosing [value if false, value if true][<test - truthy value]()
+            log_type = [NodeLog.PROJECT_REGISTERED, NodeLog.PREREG_CHALLENGE][
+                "Prereg" in draft.registration_schema.name]
```

Comment:
I note that in 6c224333abf817be99fb4db7c251472662ab2135 we lost the lambda wrapping ... except that the comment wasn't updated. ;-)